mousestyles/dynamics/tests/test_dynamics.py

```diff
@@ -98,49 +98,95 @@ def test_get_prob_matrix_small_interval():
     assert example[0, 2] == 0.2
     assert example[1, 2] == 1.
     assert sum(example[0, :]) == 1.
-
```

Comment:
Did you made any modification to Chenyu's test code? or is this deletion simply due to rebase?